plugins/lookup/passwordstore.py

```diff
@@ -77,54 +79,89 @@
           - warn
           - empty
           - create
+      lock:
+        description:
+          - How to synchronize operations.
+          - The default of C(write) only synchronizes write operations.
+          - C(readwrite) synchronizes all operations (including read). This makes sure that gpg-agent is never called in parallel.
+          - C(none) does not do any synchronization.
+        ini:
+          - section: passwordstore_lookup
+            key: lock
+        type: str
+        default: write
+        choices:
+          - readwrite
+          - write
+          - none
+        version_added: 4.5.0
+      locktimeout:
+        description:
+          - Lock timeout applied when I(lock) is not C(none).
+          - Time with a unit suffix, "s", "m", "h" for seconds, minutes, hours, e.g., C(900s) equals C(15m).
```

Comment:
```suggestion
          - Time with a unit suffix, C(s), C(m), C(h) for seconds, minutes, and hours, respectively. For example, C(900s) equals C(15m).
```

Reply:
I used quotes there in purpose (for readability and to distinguish it from the defined values and examples, as in `unit suffixes like “mm”, “cm”, e.g., C(20mm))`

Reply:
In the spirit of wrapping up this contribution without further ado, I adopted your preferred wording.